test/async.js

```diff
@@ -0,0 +1,220 @@
+import crypto from 'crypto';
+import fs from 'fs';
+import path from 'path';
+
+import bufferEquals from 'buffer-equals';
+import pify from 'pify';
+import rewire from 'rewire';
+import rimraf from 'rimraf';
+import test from 'ava';
+import uuid from 'uuid';
+
+import cpFile from '../';
+import {assertDateEqual} from './_assert';
+
+const fsP = pify(fs, Promise);
```

Comment:
You aren't using a `Promise` polyfill anywhere though. You're only passing the built in `Promise` (which is already used by `pify`).